pkg/server/types.go

```diff
@@ -41,5 +42,6 @@ type BackendMapper struct {
 
 // AccessConfig represents the configuration format for cluster access config via backend mode.
 type BackendModeConfig struct {
+	dynamicfile.Meta
```

Comment:
i dont' feel that this config needs teh extra fields. Could you tell more why the new fields are relevant for other backend modes

Reply:
The access config file is propagated via CAM, and so the extra fields are needed to calculate the latency and version of that change, regardless if dynamic file was enabled or not. I did remove the nested struct field as it's not specific to dynamicfile.